src/System.Windows.Forms.Design/tests/UnitTests/ButtonBaseDesignerTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+using System.Collections;
+
+namespace System.Windows.Forms.Design.Tests;
+
+public sealed class ButtonBaseDesignerTests
+{
+    [Fact]
+    public void ButtonBaseDesigner_Constructor_Initialize_AutoResizeHandles()
+    {
+        using ButtonBaseDesigner buttonBaseDesigner = new();
+        bool autoResizeHandles = buttonBaseDesigner.AutoResizeHandles;
+
+        buttonBaseDesigner.Should().NotBeNull();
+        autoResizeHandles.Should().Be(true);
+    }
+
+    public static IEnumerable<object[]> IDictionary_TestData()
+    {
+        yield return new object[] { null! };
+        yield return new object[] { new Dictionary<string, object>() };
+    }
```

Comment:
Are you getting errors if you avoid the `!`?
```suggestion
    public static IEnumerable<object?[]> IDictionary_TestData()
    {
        yield return new object[] { null };
        yield return new object[] { new Dictionary<string, object>() };
    }
```

Reply:
> Are you getting errors if you avoid the `!`?

Yes, if removing ! here,  there will be a warning `Cannot convert null literal to non-nullable reference type.	System.Windows.Forms.Design.Tests	
`